### ISSUE TITLE
Nanopolish consensus for reads basecalled not with Metrichor

### ISSUE THREAD

**Issue by damlasenol** [jts/nanopolish#193]
Hello,
I want to find the consensus sequence for the draft assembly of reads not basecalled with Metrichor, but with Deepnano. After modifying the headers of fasta file, I ran the bwa commands without any problem. After running the nanopolish variants --consensus command for each 50kb segment with 10 threads, I am getting these kind of warnings and each 50kb part finishes in 22minutes without outputting any variants: (below output is for the segment 350000-400000)
##fileformat=VCFv4.2
##INFO=<ID=TotalReads,Number=1,Type=Integer,Description="The number of event-space reads used to call the variant">
##INFO=<ID=SupportFraction,Number=1,Type=Float,Description="The fraction of event-space reads that support the variant">
##INFO=<ID=BaseCalledReadsWithVariant,Number=1,Type=Integer,Description="The number of base-space reads that support the variant">
##INFO=<ID=BaseCalledFraction,Number=1,Type=Float,Description="The fraction of base-space reads that support the variant">
##INFO=<ID=AlleleCount,Number=1,Type=Integer,Description="The inferred number of copies of the allele">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
#CHROM  POS     ID      REF     ALT     QUAL    FILTER  INFO    FORMAT  sample
Warning: 110719 variants in span, region not called [349970 400230]
[post-run summary] total reads: 2460 unparseable: 0 qc fail: 0 could not calibrate: 0
This does not happen with consensus calling for the draft assembly with Metrichor reads. Should I add another parameter or am I omitting something?
Thank you,
Damla

**Comment by jts** [comment]
Hi Damla,

If I understand correctly you modified the headers of the fasta file that deepnano produced to include the fast5 paths? Unfortunately this will not work as nanopolish requires the basecalls in the fast5 file to match the aligned reads in the bam. Since these will mismatch in your case (the fast5 will contain metrichor or albacore basecalls, but the bam contains deepnano basecalls), nanopolish will fail. 

In short, you can't use deepnano basecalled reads with nanopolish, sorry.

Jared

**Comment by damlasenol** [comment]
Hi Jared,

Thank you for the clarification. Then, I cannot use nanopolish for the draft assemblies of deepnano, nanocall or nanonet reads, am I right?

Damla

**Comment by jts** [comment]
That's right.

**Comment by damlasenol** [comment]
Thank you!
